src/controllers/notifications.js

```diff
@@ -1,67 +0,0 @@
-'use strict'
```

Comment:
By checking the expiry: resending a new notification request after a transaction should have expired.